content/docs/1.0.0/iofogctl/iofogctl.md

```diff
@@ -0,0 +1,209 @@
+# ioFog Unified Command Line Interface
+
+In this tutorial, we will go through basic functionality of `iofogctl`. This tutorial will get us up and running with iofogctl and show how to deploy and operate a live cluster.
+
+`iofogctl` is a CLI tool for installation, configuration, and operation of ioFog Edge Compute Networks (ECNs).
+
+It can be used to remotely manage multiple different clusters from a single host. It is built for an ioFog user and a DevOps engineering wanting to manage ioFog clusters.
+
+## Prerequisites
+
+The following must be installed and configured before performing bootstrap:
+
+- Go 1.12.1+ ([installation instructions](https://golang.org/doc/install))
```

Comment:
Only required if you wish to install from source

Reply:
Yea don't think we need to show them this for tutorial. Its a contributing.md kinda topic

Reply:
In order to use iofogctl for non-local deployments, you will need to provide your own infrastructure. This means you will need to provision a Kubernetes cluster and a set of edge nodes yourself. You can use our Platform repository to assist this infrastructure deployment: http://github.com/eclipse-iofog/platform.git
| Component        | Prerequisites for new deployment           | Prerequisites for connecting to existing deployment  |
| ------------- |-------------| -----|
| Controller  | Valid Kubernetes config file (e.g. usually found in ~/.kube/config)      | Password of ioFog user on existing Controller |
| | | Email of ioFog user on existing Controller |
| | |Valid Kubernetes config file (e.g. usually found in ~/.kube/config) or IP address of Controller |
| Agent | RSA priv/pub key pair that can be used to SSH into the respective machine      |    |
| | A user that is in sudo group (e.g. usermod -aG sudo $USER) on the machine  | |
| | Users in group sudo can sudo without a password (use su visudo to edit sudoers file to have %sudo ALL=(ALL) NOPASSWD:ALL) on the machine | N/A - connecting to existing Controller will automatically connect you to provisioned Agents |